components/web/goaccess/goaccess.p5m

```diff
@@ -23,13 +24,15 @@ set name=org.opensolaris.consolidation value=$(CONSOLIDATION)
 license $(COMPONENT_LICENSE_FILE) license='$(COMPONENT_LICENSE)'
 
 file path=etc/goaccess.conf preserve=true
```

Comment:
Did you run "gmake publish"? On my machine it fails:
Couldn't find 'etc/goaccess.conf' in any of the specified search directories:
	/oibuild/oi-userland/components/web/goaccess
	/oibuild/oi-userland/components/web/goaccess/build
	/oibuild/oi-userland/components/web/goaccess/build/prototype/i386
	/oibuild/oi-userland/components/web/goaccess/build/prototype/i386/mangled
	/oibuild/oi-userland/components/web/goaccess/goaccess-1.4
gmake: *** [/oibuild/oi-userland/make-rules/ips.mk:434: /oibuild/oi-userland/components/web/goaccess/build/manifest-i386-goaccess.depend] Error 1

Reply:
So, your idea with saving the old goaccess.conf file is fine but doesn't work as it is now. I propose to remove the line and warn possible users over the mailing list resp. release notes.
It is probably possible to do what you intended to do by means of bypass-generate  but I wouldn't do that here.